test-app/openapi/v1/openapi.json

```diff
@@ -14,7 +14,9 @@
         "operationId": "testBasicAuth",
         "security": [
           {
-            "basic_auth": []
+            "basic_auth": [
+
+            ]
```

Comment:
any idea why the three newlines have been added in this file to the empty arrays?

Reply:
Don't know really :confused: Didn't find anything I've removed in this PR that would cause this.

Reply:
I see the same happened in this commit: https://github.com/rswag/rswag/pull/675/commits/d65f4fe4db17550809d462fd35d6edc2010dc6d2, so I'm suspecing some change in dependency that rswag uses has caused this.